app/src/main/java/src/LinedMove.java

```diff
@@ -1,79 +1,85 @@
 package src;
 
 import src.Pieces.Piece;
-import java.awt.*;
 import java.util.ArrayList;
 import java.util.List;
 
 public class LinedMove implements MovePiece {
-    List<Position> movePoints = new ArrayList<Position>();
 
     @Override
     public List<Position> getPossibleMoves(Piece piece) {
-        //Down
-        int posY = piece.getInitPosY();
-        int posX = piece.getInitPosX();
-        for (int i = posX + 1; i < Chessboard.DIMENSION; i++) {
-            if (!isAccessiblePosition(i, posY, piece.getColorWhite())) {
-                break;
-            }
+        List<Position> movePositions = new ArrayList<>();
+        int xPos = piece.getInitPosX();
+        int yPos = piece.getInitPosY();
+
+        for( int i = xPos + 1; i < Chessboard.DIMENSION; i++) {
+            if(isSpaceEmpty(Chessboard.board[yPos][i])) {
+                movePositions.add(new Position(i, yPos));
+            }else if(isSpaceWithEnemy(Chessboard.board[yPos][i], piece.getColorWhite())) {
```

Comment:
Care with spaces, leave a space before the }_else and after the conditional if_()